addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -488,6 +530,16 @@ var FormController = BasicController.extend({
         event.stopPropagation();
         this.model.freezeOrder(event.data.id);
     },
+    _onFocusControlButton:function(e) {
```

Comment:
missing docstring (in particular the type of e), should be tagged private

Reply:
done